README.md

```diff
@@ -25,7 +25,23 @@ Sample `.pre-commit-config.yaml`:
 ## cli
 
 ```console
+$ sync-pre-commit-deps -h
```

Comment:
personally I usually leave this off and recommend the user run `--help` themselves.  since this is almost always out of date / out of sync

Reply:
sure